Optimize abstracts of Mng AKs (CQA) (#4960)
theforeman/foreman-documentation: guides/common/modules/con_managing-activation-keys.adoc

Author: Does it make sense to use AKs for other OSs than Enterprise Linux?

Reviewer: Yes, all supported Client OS use AK to consume content from Foreman+Katello.

= Managing activation keys

[role="_abstract"]

> Activation keys automate host configuration during registration to {Project} by assigning content view environments, enabling repositories, and defining system purpose.

Reviewer: system purpose is RHEL-only:

```suggestion
ifdef::katello,almalinux,amazon_linux,centos,debian,oracle_linux,rocky_linux,suse_linux_enterprise_server,ubuntu[]
Activation keys automate host configuration during registration to {Project} by assigning content view environments and enabling repositories.
endif::[]
ifdef::satellite,red_hat_enterprise_linux[]
Activation keys automate host configuration during registration to {Project} by assigning content view environments, enabling repositories, and defining system purpose.
endif::[]
```

> You can create and configure keys in the web UI or with Hammer CLI and apply best practices when using multiple keys.

Author: ```suggestion
Activation keys automate host configuration during registration to {Project} by assigning content view environments, enabling repositories, and defining system purpose.
Create and configure your activation keys to scale consistent configuration across many hosts.
```

> Create and configure your activation keys to scale consistent configuration across many hosts.

Reviewer: I would prefer "content" (or "content access") over "configuration" because users typically use Ansible/Puppet/Salt to enforce a consistent configuration.

Author: ```suggestion
Create and configure your activation keys to scale consistent content access across many hosts.
```
